### PR TITLE
feat: added new property in user preferences for welcome message dismissal

### PR DESCRIPTION
**Pull request by birtony**
Fixes https://github.com/trustbloc/wallet/issues/1400

Signed-off-by: Anton Biriukov <anton.biriukov@securekey.com>

### CODE REVIEW

File: cmd/wallet-js-sdk/src/user/wallet-user.js [hunk 1]
```diff
@@ -189,6 +190,7 @@ export class WalletUser {
       controller = "",
       verificationMethod = "",
       proofType = "",
+      welcomeMessageDismissed = false,
```
Comment (rolsonquadras) [inline cmd/wallet-js-sdk/src/user/wallet-user.js:193]: imo `skipWelcomeMsg` would be more concise.

**Review by rolsonquadras — APPROVED** [review]

File: cmd/wallet-js-sdk/src/user/wallet-user.js (hunk 1 above)
Comment (sudeshrshetty) [reply cmd/wallet-js-sdk/src/user/wallet-user.js:193]: yes, some generic name, so that it can be used community for other purposes too.

Comment (birtony) [reply cmd/wallet-js-sdk/src/user/wallet-user.js:193]: Should I make it optional as well @rolsonquadras @sudeshrshetty?

Comment (sudeshrshetty) [reply cmd/wallet-js-sdk/src/user/wallet-user.js:193]: it is optional already since u r setting default value, all fields I see here are optional.

Comment (birtony) [reply cmd/wallet-js-sdk/src/user/wallet-user.js:193]: Yes, sure. I mean in the parameters description I'd add `(optional)` as well

Comment (sudeshrshetty) [reply cmd/wallet-js-sdk/src/user/wallet-user.js]: yes

Comment (birtony) [reply cmd/wallet-js-sdk/src/user/wallet-user.js]: Updated

**Review by sudeshrshetty — APPROVED** [review]

**Pull request merged**